keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ApplicationSummary.kt

```diff
@@ -0,0 +1,14 @@
+package com.netflix.spinnaker.keel.core.api
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder(value = ["name", "application", "serviceAccount", "apiVersion", "isPaused"])
+data class ApplicationSummary(
+  val name: String,
+  val application: String,
```

Comment:
What's the difference between `name` and `application`?

Reply:
That's the delivery config name, right? Maybe be just need to clarify that?

Reply:
changed `name` to `deliveryConfigName`